pkg/cmd/server/kubernetes/node/options/options.go

```diff
@@ -88,9 +87,6 @@ func ComputeKubeletFlagsAsMap(startingArgs map[string][]string, options configap
 	if network.IsOpenShiftNetworkPlugin(options.NetworkConfig.NetworkPluginName) {
 		// SDN plugin pod setup/teardown is implemented as a CNI plugin
 		setIfUnset(args, "network-plugin", kubeletcni.CNIPluginName)
-		setIfUnset(args, "cni-conf-dir", kubeletcni.DefaultNetDir)
-		setIfUnset(args, "cni-bin-dir", kubeletcni.DefaultCNIDir)
-		setIfUnset(args, "hairpin-mode", kubeletconfig.HairpinNone)
```

Comment:
Instead of hard-coding  cniDirPath="/etc/cni/net.d" in pkg/network/node/node.go, 
we can actually derive the CNI conf dir:
```
  cniDirPath = kubeletArguments["cni-conf-dir"][0] if present, otherwise kubeletcni.DefaultNetDir
```
By doing this, we do not have to change anything in openshift-sdn when upstream changes these default values.